packages/@sanity/desk-tool/src/panes/documentPane/timeline/timeline.tsx

```diff
@@ -1,15 +1,14 @@
 // @todo: remove the following line when part imports has been removed from this file
 ///<reference types="@sanity/types/parts" />
 
-import React, {forwardRef, useCallback, useEffect, useRef, useState} from 'react'
+import React, {useCallback, useEffect, useRef, useState} from 'react'
 import {Chunk} from '@sanity/field/diff'
-import {Stack, Text} from '@sanity/ui'
+import {Box, Menu, Stack, Text} from '@sanity/ui'
 import Spinner from 'part:@sanity/components/loading/spinner'
```

Comment:
This `Spinner` should be replaced by the `import {Spinner} from '@sanity/ui'`

Reply:
Removed 🎉